### ISSUE TITLE
Metabase doesn't work with PostgreSQL 10

### ISSUE THREAD

**Issue by und3fined** [metabase/metabase#6120]
### Bugs
- Your operating system: `Linux`
- Java version: ```java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)```
- Metabase version: `0.26.1`
- Metabase hosting environment:   `Red Hat Enterprise Linux Server release 7.4 (Maipo)`
- Metabase internal database: `PostgreSQL 10.0 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-16), 64-bit`

- *Repeatable steps to reproduce the issue*

```
[root@ip-10-0-0-1 metabase-0.26.1]# java -jar metabase.jar
10-08 12:09:27 INFO metabase.util :: Loading Metabase...
10-08 12:09:36 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
10-08 12:09:58 INFO metabase.core :: Starting Metabase in STANDALONE mode
10-08 12:09:58 INFO metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

10-08 12:09:58 INFO metabase.core :: Starting Metabase version v0.26.1 (dacff52 release-0.26.1) ...
10-08 12:09:58 INFO metabase.core :: System timezone is 'UTC' ...
10-08 12:09:58 WARN metabase.driver :: No -init-driver function found for 'metabase.driver.google'
10-08 12:09:59 INFO metabase.core :: Setting up and migrating Metabase DB. Please sit tight, this may take a minute...
10-08 12:09:59 INFO metabase.db :: Verifying postgres Database Connection ...
10-08 12:09:59 INFO metabase.db :: Verify Database Connection ...  ✅
10-08 12:09:59 INFO metabase.db :: Running Database Migrations...
org.postgresql.util.PSQLException: This ResultSet is closed.
	at org.postgresql.jdbc2.AbstractJdbc2ResultSet.checkClosed(AbstractJdbc2ResultSet.java:2852)
	at org.postgresql.jdbc2.AbstractJdbc2ResultSet.setFetchSize(AbstractJdbc2ResultSet.java:1875)
	at org.postgresql.jdbc4.Jdbc4Statement.createResultSet(Jdbc4Statement.java:37)
	at org.postgresql.jdbc2.AbstractJdbc2Statement$StatementResultHandler.handleResultRows(AbstractJdbc2Statement.java:221)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1853)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:255)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:561)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:405)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.execSQLUpdate(AbstractJdbc2Connection.java:382)
	at org.postgresql.jdbc2.AbstractJdbc2Connection.getTransactionIsolation(AbstractJdbc2Connection.java:904)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:705)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:740)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
	at clojure.java.jdbc$db_transaction_STAR_.invokeStatic(jdbc.clj:697)
	at clojure.java.jdbc$db_transaction_STAR_.invoke(jdbc.clj:684)
	at metabase.db$migrate_BANG_.invokeStatic(db.clj:235)
	at metabase.db$migrate_BANG_.invoke(db.clj:218)
	at metabase.db$run_schema_migrations_BANG_.invokeStatic(db.clj:365)
	at metabase.db$run_schema_migrations_BANG_.invoke(db.clj:360)
	at metabase.db$setup_db_BANG_.invokeStatic(db.clj:383)
	at metabase.db$setup_db_BANG_.doInvoke(db.clj:376)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at metabase.core$init_BANG_.invokeStatic(core.clj:103)
	at metabase.core$init_BANG_.invoke(core.clj:82)
	at metabase.core$start_normally.invokeStatic(core.clj:187)
	at metabase.core$start_normally.invoke(core.clj:181)
	at metabase.core$_main.invokeStatic(core.clj:285)
	at metabase.core$_main.doInvoke(core.clj:280)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at metabase.core.main(Unknown Source)
10-08 12:09:59 ERROR metabase.core :: Metabase Initialization FAILED:  This ResultSet is closed.
10-08 12:09:59 INFO metabase.core :: Metabase Shutting Down ...
10-08 12:09:59 INFO metabase.core :: Metabase Shutdown COMPLETE

```

**Comment by trifle** [comment]
This will supposedly be fixed by https://github.com/metabase/metabase/commit/d23b26694056456cc370580841820dc4f55ae9a4 which is in master but not yet in a release.

**Comment by senior** [comment]
@und3fined @trifle We just released PostgreSQL 10 support with 0.26.2, if you upgrade you should see it work now.

**Comment by nurey** [comment]
The docker version doesn't seem to be detecting that 0.26.2 is out.
